wellregistry/registry/admin/monitoring_location.py

```diff
@@ -15,13 +15,164 @@
 from .fetch_from_nwis import FetchFromNwisView
 
 USGS_AGENCY_CD = 'USGS'
-
+tool_tips = [{
+		"form_field": "display_flag",
+		"tooltip": "Flag that controls whether the site will be displayed on the NGWMN Data Portal (Yes/No)"
+	},
+	{
+		"form_field": "agency",
+		"tooltip": "The name of the agency or organization that collects, owns andor manages the data. If you don't see your agency or organization in the list contact Jessica Lucido(lucido @usgs.gov)."
+	},
+	{
+		"form_field": "site_no",
+		"tooltip": "The local unique well or spring identification number or code. We will be requesting water levels and water - quality data from(Organization) -owned  managed databases by this unique well identifier.This identifier may be alphanumeric,but should not contain spaces."
+	},
+	{
+		"form_field": "site_name",
+		"tooltip": "The local name of well or spring"
+	},
+    {
+		"form_field": "country",
+		"tooltip": "The name of the country within which the well or spring resides"
+	},
+	{
+		"form_field": "state",
+		"tooltip": "The U.S. State or Territory within which the well or spring resides"
+	},
+	{
+		"form_field": "county",
+		"tooltip": "The name of the county within which the well or spring resides"
+	},
+	{
+		"form_field": "dec_lat_va",
+		"tooltip": "The site latitude, in decimal degrees to the accuracy of your measurement"
+	},
+	{
+		"form_field": "dec_long_va",
+		"tooltip": "The site longitude, in decimal degrees (negative for Western Hemisphere) to the accuracy of your measurement"
+	},
+	{
+		"form_field": "horizontal_datum",
+		"tooltip": "Horizontal reference datum for the latitude and longitude of a well (NAD83,NAD27, WGS84, etc.) If you don’t see the datum you are looking for in the list contact Jessica Lucido (jlucido@usgs.gov)"
```

Comment:
You should check with Candice as to the contact. I'm thinking it should be her and not Jessica

Reply:
fixed